.github/workflows/test.yml

```diff
@@ -37,6 +37,19 @@ jobs:
         python: ['3.8', '3.11']
         django: ['3.2', '4.2']
         wagtail: ['5.1', '5.2']
+        include:
+          - python: '3.12'
+            django: '4.2'
+            wagtail: '5.2'
+          - python: '3.12'
+            django: '4.2'
+            wagtail: '6.0'
```

Comment:
This one too

Reply:
Updated in: https://github.com/torchbox-forks/wagtail-sharing/pull/11/commits/346d5234834a566717222a1d3abf6ea33ec3248c